adoc/headers/group.h

```diff
@@ -49,6 +49,32 @@ template <int Dimensions = 1> class group {
   void parallel_for_work_item(range<Dimensions> logicalRange,
                               const WorkItemFunctionT& func) const;
 
+  // Deprecated in SYCL 2020. 
+  template <typename DataT>
+  device_event async_work_group_copy(local_ptr<DataT> dest,
+                                     global_ptr<const DataT> src,
+                                     size_t numElements) const;
+
+  // Deprecated in SYCL 2020.
+  template <typename DataT>
+  device_event async_work_group_copy(global_ptr<DataT> dest,
+                                     local_ptr<const DataT> src,
+                                     size_t numElements) const;
+
+  // Deprecated in SYCL 2020.
+  template <typename DataT>
+  device_event async_work_group_copy(local_ptr<DataT> dest,
+                                     global_ptr<const DataT> src,
+                                     size_t numElements,
+                                     size_t srcStride) const;
+
+  // Deprecated in SYCL 2020.
+  template <typename DataT>
+  device_event async_work_group_copy(global_ptr<DataT> dest,
+                                     local_ptr<const DataT> src,
+                                     size_t numElements,
+                                     size_t destStride) const;
+
```

Comment:
These new overloads should also be added to Table 118 "Member functions for the group class" (section 4.9.1.7).  I think the description could be simplified like this, though:

> Deprecated in SYCL 2020.  Has the same effect as the overload taking `decorated_local_ptr` and `decorated_global_ptr` except that the `dest` and `src` parameters have `access::decorated::legacy`.

Reply:
Good catch, fixed in https://github.com/KhronosGroup/SYCL-Docs/pull/432/commits/b84963dbb7a353890317fcaa01c8c172a727ae68.